modules/dnn/src/tensorflow/tf_importer.cpp

```diff
@@ -1468,6 +1468,8 @@ void TFImporter::populateNet(Net dstNet)
             int end_mask = getLayerAttr(layer, "end_mask").i();
             for (int i = 0; i < num; ++i)
             {
+                if (ends.at<int>(i) <= 0)
```

Comment:
Please add a test for this

Reply:
@dkurt , I have added the test, model and data.